config/api/routes/lock.php

```diff
@@ -44,28 +22,6 @@
             if ($lock = $this->parent($path)->lock()) {
                 return $lock->remove();
             }
-
-            throw new Exception([
-                'key'      => 'lock.notImplemented',
-                'httpCode' => 501
-            ]);
-        }
-    ],
-    [
-        'pattern' => '(:all)/unlock',
```

Comment:
Same as above. I wonder if we should actually deprecate those. We haven't documented them at all right now and thus we could say we deprecate them without harm, but the big question remains if we want to support locking via the API in the future or not, or if we keep this a private API.

Reply:
We can do that. Tbh, I think this is a very theoretical discussion - I don't think anyone has ever implemented content locking for virtual pages due to the state of our code and docs. I'd also bet that there's no API use besides us here. But sure, we can be on the safe side.